spec/rspec/core/source/syntax_highlighter_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 class RSpec::Core::Source
   RSpec.describe SyntaxHighlighter do
-    let(:config)      { RSpec::Core::Configuration.new.tap { |c| c.color = true } }
+    let(:config)      { RSpec::Core::Configuration.new.tap { |config| config.no_color = false } }
```

Comment:
Does `no_color = false` actually do anything?  Is it necessary to modify the config here at all?